buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -42,27 +42,27 @@ object Config {
     //   https://detekt.github.io/detekt/performance.html
     //   https://detekt.github.io/detekt/style.html
     // Release info: https://github.com/detekt/detekt/releases
-    const val DETEKT_VERSION = "1.19.0"
+    const val DETEKT_VERSION = "1.21.0"
     // https://android-developers.googleblog.com/2021/09/accelerated-kotlin-build-times-with.html
     // Supported libs: https://android-developers.googleblog.com/2021/09/accelerated-kotlin-build-times-with.html
     // https://kotlinlang.org/docs/ksp-overview.html
     // https://kotlinlang.org/docs/ksp-quickstart.html
     // https://github.com/google/ksp/releases
     // TODO: First part of version should match version of kotlin - see https://github.com/google/ksp/blob/main/docs/faq.md
-    const val KSP_VERSION = "1.7.0-1.0.6"
+    const val KSP_VERSION = "1.7.20-1.0.8"
 
     /**
      * Called from root project buildscript block in the project root build.gradle.kts
      */
     object BuildScriptPlugins {
         // https://developer.android.com/studio/releases/gradle-plugin
         // TODO: Update corresponding buildSrc/build.gradle.kts value when updating this version!
-        const val ANDROID_GRADLE = "com.android.tools.build:gradle:7.2.0"
+        const val ANDROID_GRADLE = "com.android.tools.build:gradle:7.3.0"
```

Comment:
@LuisRocketRangel In reference to the previous comment about this dependency `com.android.tools.build:gradle` in the build.gradle.kts, I believe this is the one that should match up